### PR TITLE
Add .gitignore

### PR DESCRIPTION
**Pull request by Vthechamp22**
Add a .gitignore file!

### CODE REVIEW

**Review by janine9vn — APPROVED** [review]
Looks good!

**Comment by busterbeam** [comment]
good this got approved, thanks janine9vn

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+coverage.xml
+*.cover
+*.py,cover
+.hypothesis/
+.pytest_cache/
+cover/
+
+# Translations
+*.mo
+*.pot
```
Comment (Bluenix2) [inline .gitignore:60]: Will our project be translated?

Comment (Bluenix2) [reply .gitignore:60]: Hmmm...

Comment (busterbeam) [reply .gitignore:60]: no translations!!

**Review by poke5352 — APPROVED** [review]

**Comment by pedro-anthony** [comment]
looks good enough.

**Comment by roogla** [comment]
Looks good.

**Review by busterbeam — APPROVED** [review]
I approve this change!

**Comment by urnniitese** [comment]
Awesome looks good!

**Review by Oreo4102004 — COMMENTED** [review]
*.oreoxD

**Comment by JoeNibe** [comment]
!!

**Review by Arnav-2004 — APPROVED** [review]
LGTM!

**Review by Oreo4102004 — APPROVED** [review]
looksgood

**Review by 9pa — APPROVED** [review]

**Comment by Bluenix2** [comment]
@pedro-anthony and @roogla, you most likely actually want to use the review feature. Click on the Files Changed tab all the way to the right. Then you'll see a green Review Changes button, there you can choose if you'd like to Approve, Comment or Request Changes.

**Review by roogla — COMMENTED** [review]
Maybe I reviewed the changes for the changed mind.

**Review by janine9vn — CHANGES_REQUESTED** [review]
Hey, I think we're missing a file. Please add .something

**Review by busterbeam — APPROVED** [review]

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+.pytype/
+
+# Cython debug symbols
+cython_debug/
+
+# End of https://www.toptal.com/developers/gitignore/api/python
+
+
+# Custom files and folders
+.logs/
```
Comment (Bluenix2) [inline .gitignore:148]: There's two empty lines here, I suggest we remove those..
```suggestion
# Pyre type checker
.pyre/

# pytype static type analyzer
.pytype/

# Cython debug symbols
cython_debug/

# End of https://www.toptal.com/developers/gitignore/api/python

# Custom files and folders
.logs/
```

**Comment by Vthechamp22** [comment]
:partying_face:

**Pull request merged**